src/Expensify.js

```diff
@@ -90,6 +91,13 @@ function Expensify(props) {
 
     const isAuthenticated = useMemo(() => Boolean(lodashGet(props.session, 'authToken', null)), [props.session]);
 
+    useEffect(() => {
+        if (Platform.OS !== 'ios') return;
```

Comment:
According to Expensify's rules, we're supposed to avoid Platform.OS calls. I think this has to be in a dedicated component that has iOS and non-iOS versions.